install_config/install/advanced_install.adoc

```diff
@@ -776,15 +776,65 @@ openshift_examples_modify_imagestreams=true
 
 |`*openshift_docker_additional_registries*`
 |Specify the additional registry or registries.
+
+|`*openshift_cockpit_deployer_prefix*`
+|Specify the URL and path to the directory where the image is located.
+
+|`*openshift_web_console_prefix*`
+|The prefix for the component images. For example, with
+ifdef::openshift-origin[]
+`openshift/origin-web-console:v3.9`, set prefix `openshift/origin-`.
+endif::[]
+ifdef::openshift-enterprise[]
+`openshift3/ose-web-console:v3.9`, set prefix `openshift3/ose-`.
+endif::[]
+
+|`*openshift_template_service_broker_image_prefix*`
+|
+
+|`*openshift_service_catalog_image_prefix*`
+|For use if you have the service catalog enabled.
+
+|`*ansible_service_broker_image_prefix*`
```

Comment:
How about this:
Specify the prefix for the ansible service broker component image. For example:
example.com/openshift3/ose-